### ISSUE TITLE
missing cri-o package in repo for ubuntu 18.04 / cri-o 1.22

### ISSUE THREAD

**Issue by xinau** [cri-o/cri-o#5299]
**Description**

While trying to install the latest cri-o version using the provided mirrors and installation guide. I wasn't able to find any package named `cri-o`. After further investigation I've taken a look with my browser in the [repo](https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable:/cri-o:/1.22/xUbuntu_18.04/) only to notice the lack of a cri-o package there as well. As version 1.21 contains this package and a quick look at the release notes don't seem to mention the deprecation of Ubuntu 18.04 as a platform, this seems to me like a bug.

**Steps to reproduce the issue:**

```bash
CRI_O_OS=xUbuntu_18.04
CRI_O_VERSION=1.22
echo "deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable:/cri-o:/$CRI_O_VERSION/$CRI_O_OS/ /" | sudo tee /etc/apt/sources.list.d/devel:kubic:libcontainers:stable:cri-o:$CR_IO_VERSION.list
curl -L https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable:cri-o:$CRI_O_VERSION/$CRI_O_OS/Release.key | sudo apt-key add -
sudo apt update && sudo apt search cri-o
# Sorting... Done
# Full Text Search... Done
```

**Comment by msschl** [comment]
There are some more packages missing...

Looking at http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable:/cri-o:/1.20/xUbuntu_20.04/InRelease
the latest release for ubuntu 20.04 is cri-o version: 1.20.3~0. However, the latest release is 1.20.5 released on the 14.09.2021

**Comment by haircommander** [comment]
Sorry folks I have trouble maintaining all of the different versions for all the different distributions people want. I believe 1.20 and 1.21 are fixed, 1.19 is nearly up to date, and I'm looking into the 1.22 issues. thanks for the patience

**Comment by haircommander** [comment]
18.04 and 1.22/1.21/1.20/1.19 are fixed, thanks for the patience folks! I am closing this as such :) thanks for using cri-o!
